cmd/clusterctl/client/config_test.go

```diff
@@ -849,9 +866,9 @@ func newFakeClientWithoutCluster(configClient config.Client) *fakeClient {
 	}
 
 	var err error
-	fake.internalClient, err = newClusterctlClient("fake-config",
+	fake.internalClient, err = newClusterctlClient(context.TODO(), "fake-config",
```

Comment:
```suggestion
	fake.internalClient, err = newClusterctlClient(context.Background(), "fake-config",
```

Should this be `context.Background()`?